### DISCUSSION TITLE
Copy text to clipboard without AI interaction

### DISCUSSION THREAD

**Discussion by zheroz00** [ILikeAI/AlwaysReddy#31] [Ideas]
Hey Josh, I used to have an app where I could use Whisper globally across my system, but not for interacting with AI like yours does. It simply recorded your voice, translated it to text, and copied it to your clipboard (and pasted if your cursor was in a text field). Would it be possible to have a function like that as a separate key shortcut? So basically, if you just wanted to copy some text to the clipboard, you could simply click the new keyboard combo, say what you got to say, have it copied to clipboard. Would that be something that could be implemented when you have time? Thanks.

**Comment by ILikeAI** [comment]
This is a great idea, yep that could for sure be added :)

**Comment by ILikeAI** [comment]
Kinda makes me wonder about having another hotkey that runs whatever is in your clipboard through TTS as well, sometimes I have a chain of messages I want to catch up on but I dont want to read them all, could be handy for that. What do you think?

**Reply by zheroz00** [reply]
That sounds like a great idea!
